qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -345,6 +350,14 @@ def set_dictionary_language(self):
         self._profile.setSpellCheckLanguages(filenames)
         self._profile.setSpellCheckEnabled(bool(filenames))
 
+    def disable_persistent_permissions_policy(self):
+        """Disable webengine's permission persistence."""
+        pyqt_version = utils.VersionNumber.parse(version.PYQT_WEBENGINE_VERSION_STR)
+        if pyqt_version >= utils.VersionNumber(6, 8):
```

Comment:
:disappointed: I didn't know it could be the other way around too.

I don't have a super high confidence in all the layers fitting together right with all the different versions and patches floating around etc. I would prefer something more explicit where we could do `if pyqt_webengine > 6.8 and webengine > 6.8: ...` and then we would know to expect it and if it wasn't there it would explode loudly. Otherwise I worry it could swallow errors in some way.

But it's not a big deal.

Reply:
Yeah I'm really not a fan either, but unfortunately it's a situation we've seen before in the wild... I've been thinking about adding a `utils.qt_features` (and maybe `utils.qtwe_features`) that has different `def has_persistent_permissions_policy() -> bool:` etc. functions, and tries to kinda figure out those things autotools `./configure` style. I'm really not sure if it'd suck less, especially given it reminds me of autotools. :sweat_smile: 

The other solution I can think of would be to refuse running if Qt/PyQt doesn't fit together (or at least if PyQt is older than Qt), but that seems like a recipe for trouble as well.